unit-tests/src/test/scala/java/util/PropertiesTest.scala

```diff
@@ -358,32 +349,51 @@ class PropertiesTest {
                  prop.store(null: OutputStream, ""))
   }
 
+  def store(props: Properties, header: String = ""): ByteArrayOutputStream = {
+    val out = new ByteArrayOutputStream()
+    props.store(out, header)
+    out.close()
+    out
+  }
+
+  def storeWriter(props: Properties,
+                  header: String = ""): ByteArrayOutputStream = {
+    val out = new ByteArrayOutputStream()
+    props.store(new OutputStreamWriter(out), header)
+    out.close()
+    out
+  }
+
+  def load(out: ByteArrayOutputStream): Properties =
+    load(new ByteArrayInputStream(out.toByteArray))
+
+  def load(in: ByteArrayInputStream): Properties = {
+    val props = new Properties()
+    props.load(in)
+    in.close()
+    props
+  }
+
+  def assertAll(props1: Properties, props2: Properties): Unit = {
```

Comment:
Consider using more descriptive names:
```suggestion
  def assertAll(expected: Properties, actual: Properties): Unit = {
```

Reply:
Done, thank you.